website/content/docs/commands/token/capabilities.mdx

```diff
@@ -8,43 +8,68 @@ description: |-
 
 # token capabilities
 
-The `token capabilities` command fetches the capabilities of a token for a given
-path.
+List the capabilities of a token.
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault token capabilities [flags] <token> <path>
+
+$ vault token capabilities [-help | -h]
+```
+
+</CodeBlockConfig>
+
+## Description
+
+`token capabilities` command the capabilities of a token for a given path.
 
 If you pass a token value as an argument, this command uses the
 `/sys/capabilities` endpoint and permission. In the absence of an explicit token
 value, this command uses the `/sys/capabilities-self` endpoint and permission
 with the locally authenticated token.
 
+## Command arguments
```

Comment:
The last part of the description should be a Tip aside with the related API endpoints (or "None").